### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by cybernuki**

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,30 @@ def reload(self):
...
+        None if not found
+
+        cls (class): name of the class
+        id  (str): id of the object
+        """
+        if not cls or not id:
+            return None
+
+        objects = self.all(cls).values()
+        for obj in objects:
```
Comment (francisco0522) [inline models/engine/db_storage.py:90]: What does obj mean ?, you should put a more significant name to the variable

File: tests/test_models/test_engine/test_db_storage.py [hunk 2]
```diff
@@ -1,88 +1,196 @@
...
+
+        for cls in classes.values():
+            tables_elements += self.storage.count(cls)
+        self.assertEqual(all_elements, tables_elements)
+
+    @unittest.skipIf(type(models.storage) == FileStorage,
+                     "Testing FileStorage")
+    def test_get(self):
+        """ Test get method. """
+        for obj in self.storage.all().values():
```
Comment (francisco0522) [inline tests/test_models/test_engine/test_db_storage.py:190]: What does obj mean ?, you should put a more significant name to the variable

**Review by francisco0522 — COMMENTED** [review]
Please check the name of the variables to make them meaningful with their role

File: models/engine/db_storage.py (hunk 1 above)
Comment (cybernuki) [reply models/engine/db_storage.py:90]: Yeah you're right.

**Pull request merged**